Ch05/Ch05.md

```diff
@@ -1,42 +1,42 @@
+RavenDB supports a dedicated batch processing mode, using the notion of subscriptions. A subscription is simply a way to register a certain criteria on the
+database and have the database send us all the documents that match this criteria. An example of this might be, "Give me all the support calls." So far, this sounds very much
 like the streaming feature, which we covered in the previous chapter. However, subscriptions are _meant_ for batch processing. Instead of just getting all 
-the data that match the criteria, the idea is that we'll keep the subscription open forever.
+the data that matches the criteria, the idea is that we'll keep the subscription open forever.
 
-The first stage in the subscription is to send you all the existing data in the data that matches your query. Depending on your data size, this can take a 
-while. This is done in batches, and RavenDB will only proceed to send the next batch when the current one has already been acknowledge as successuflly
+The first stage in the subscription is to send all the existing data that matches the query. Depending on the amount of data, this can take a 
+while. It's done in batches, and RavenDB will only proceed to send the next batch when the current one has already been acknowledged as successfully
 processed. 
 
-The second stage in the lifetime of a subscription is when it is done going through all the data that is already in the database. This is where things 
-start to get really interesting. At this point, the subscription _isn't_ done. Instead, it is kept alive and wait around for a doucment that matches
-it criteria to arrive. When that happens, the server will immediately send that document to the subscription. In other words, not only did we get batch
-processing, but in many cases, we have _live_ batch processes. 
+The second stage in the lifetime of a subscription is when it's done going through all the data that's already in the database. This is where things 
+start to get really interesting. At this point, the subscription _isn't_ done. Instead, it's kept alive and waiting for a document that matches
+its criteria to arrive. When that happens, the server will immediately send that document to the subscription. In other words, not only did we get traditional batch
+processing, but in many cases, we also have _live_ batch processes. 
 
-Instead of waiting for a nightly process to run, you can keep the subscription open and handle changes as they come in. This keep you from having to do
-polling, remember the last items that you read, etc. RavenDB's Subscription also include error handling, recovery and retries, on the fly updates, etc. 
+Instead of waiting for a nightly process to run, you can keep the subscription open and handle changes as they come in. This keeps you from having to do things like 
+polling and remembering the last items that you read. RavenDB's subscription also includes error handling, recovery and retries and on-the-fly updates. 
```

Comment:
I interpreted your original text as linking "recovery and retries" as one item in a list of three (1. error handling, 2. recovery and retries, and 3. on-the-fly updates). (Well, and 4. etc., but I was trying to cut down on your number of "etc."s.) If "recovery and retries" are two separate items, then by all means, yes--definitely correct it as you specify in your comment. I think it would read better without the multiple "and"s.

Reply:
fixed